README.md

```diff
@@ -304,6 +304,8 @@ If you want to change your app entrypoint (for example, from `index.js` to `inde
 
 ### How to launch and debug a React Native for Windows application
 
+Before launching and debugging a React Native for Windows application, please make sure that your development environment is configured properly in accordance with [the official system requirements](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies).
+
 You can debug UWP and WPF React Native for Windows applications by changing the `platform` in your `launch.json` configuration:
```

Comment:
we don't have WPF RNW anymore, consider removing it

Reply:
Thanks for the comment, I opened [the issue](https://github.com/microsoft/vscode-react-native/issues/1515) based on it. We'll consider removing support of WPF platform for RNW in that issue.